configuration.rst

```diff
@@ -1,136 +1,158 @@
+Symfony newcomers. However, you'll get used to them quickly and you rarely need
+to change these files after package installation
 
-            <!-- config/packages/framework.xml -->
-            <?xml version="1.0" encoding="UTF-8" ?>
-            <container xmlns="http://symfony.com/schema/dic/services"
-                xmlns:framework="http://symfony.com/schema/dic/framework"
-                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd
-                    http://symfony.com/schema/dic/framework https://symfony.com/schema/dic/framework/framework-1.0.xsd"
-            >
-                <framework:config secret="%env(APP_SECRET)%">
-                    <!--<framework:csrf-protection enabled="true"/>-->
-                    <!--<framework:esi enabled="true"/>-->
-                    <!--<framework:fragments enabled="true"/>-->
-
-                    <!-- Enables session support. Note that the session will ONLY be started if you read or write from it.
-                         Remove or comment this section to explicitly disable session support. -->
-                    <framework:session/>
-
-                    <framework:php-errors log="true"/>
-                </framework:config>
-            </container>
+.. tip::
 
-    .. code-block:: php
+    To learn about all the available configuration options, check out the
+    :doc:`Symfony Configuration Reference </reference/index>` or run the
+    ``config:dump-reference`` command.
 
-        // config/packages/framework.php
-        $container->loadFromExtension('framework', [
-            'secret' => '%env(APP_SECRET)%',
-            //'default_locale' => 'en',
-            //'csrf_protection' => true,
-            //'http_method_override' => true,
-
-            // Enables session support. Note that the session will ONLY be started if you read or write from it.
-            // Remove or comment this section to explicitly disable session support.
-            'session' => [
-                'handler_id' => null,
-            ],
-            //'esi' => true,
-            //'fragments' => true,
-            'php_errors' => [
-                'log' => true,
-            ],
-        ]);
+Configuration Formats
+---------------------
 
-The top-level key (here ``framework``) references configuration for a specific
-bundle (:doc:`FrameworkBundle </reference/configuration/framework>` in this case).
+Unlike other frameworks, Symfony doesn't impose you a specific format to
+configure your applications. Symfony lets you choose between YAML, XML and PHP
+and throughout the Symfony documentation, all configuration examples will be
+shown in these three formats.
 
-.. sidebar:: Configuration Formats
+There isn't any practical difference between formats. In fact, Symfony
+transforms all of them into PHP before running the application, so there's not
```

Comment:
"transforms and caches" maybe, as transforming sounds like a performance impact?